crates/uv/tests/it/lock.rs

```diff
@@ -3791,7 +3791,7 @@ fn lock_requires_python_upper() -> Result<()> {
 /// Lock a requirement from PyPI with an exact Python bound.
 #[test]
```

Comment:
This needs the `python-patch` feature if we're going to use a specific patch version

Reply:
(which will fix Windows)